Analyzer/Resources/Init.sql

```diff
@@ -103,5 +103,272 @@ WHERE m.type = "Material";
 
 INSERT INTO types (id, name) VALUES (-1, 'Scene');
 
+CREATE TABLE addr_builds
+(
+    id INTEGER,
+    name TEXT,
+    build_target INTEGER,
+    start_time TEXT,
+    duration REAL,
+    error TEXT,
+    package_version TEXT,
+    player_version TEXT,
+    build_script TEXT,
+    result_hash TEXT,
+    type INTEGER,
+    unity_version TEXT,
+    PRIMARY KEY (id)
+);
+
+create table addr_build_bundles
```

Comment:
I'm not sure the short form will be immediately apparent to everyone "addr_build" as short for for Addressables Build.

```suggestion
create table addressables_bundles
```

I wonder if there is any easy way to make all these tables optional, e.g. only created if we actually find Addressables file?  I"m thinking now the dropdown of Tables will start with quite a long list of tables that will be all empty for some builds.  Not the end of the world but if they are self contained maybe this can be a sql file that is only run when we discover the file?

Reply:
Good call. I feel like there used to be lots of issues with SQL table name lengths so I got really stingy with names. Renamed.